tests/testthat/test_infillcrits.R

```diff
@@ -33,7 +33,7 @@ test_that("infill crits", {
       expect_true(!anyMissing(opdf$infill_ei[, c("ei","se","mean")]))
     if (!is.null(opdf$infill_cb)) {
       expect_true(!anyMissing(opdf$infill_cb[, c("se","mean","lambda")]))
-      expect_true(all(opdf$infill_cb$lambda == 2))
+      expect_true(all(opdf$infill_cb$lambda == 1))
```

Comment:
okay - you changed here that lambda should be 1 but in line 56 is specifically says `crit.cb2` which leads me to think that something is seriously not right here.